tests/spread/basic/task.yaml

```diff
@@ -0,0 +1,43 @@
+summary: Ensure that things worked
+
+execute: |
+  # for various utilities
+  . "$TESTSLIB/nested.sh"
+
+  # Start the nested UC20 VM
+  start_nested_core_vm_unit
+
+  # At this point we are able to SSH to the nested VM, so things probably worked
+  # but we should wait for snapd to finish seeding anyways, but first we need to
+  # wait for the snap command to exist
+  retry=400
+  wait=1
+  while ! execute_remote command -v snap; do
```

Comment:
I recomment to have a function in nested.sh called nested_wait_for_snap_command, so then you can reuse that in all the tests

Reply:
really good point, thanks!